packages/components/src/box-control/test/index.tsx

```diff
@@ -181,18 +238,71 @@ describe( 'BoxControl', () => {
 
 			await user.type(
 				screen.getByRole( 'textbox', {
-					name: 'Vertical',
+					name: 'Top and bottom sides',
 				} ),
 				'100'
 			);
 
 			expect(
-				screen.getByRole( 'textbox', { name: 'Vertical' } )
+				screen.getByRole( 'textbox', { name: 'Top and bottom sides' } )
 			).toHaveValue( '100' );
 			expect(
-				screen.getByRole( 'textbox', { name: 'Horizontal' } )
+				screen.getByRole( 'textbox', { name: 'Left and right sides' } )
 			).not.toHaveValue();
 		} );
+
+		it( 'should update a whole axis using a slider when value is changed when unlinked', async () => {
+			const user = userEvent.setup();
+
+			render( <Example splitOnAxis /> );
+
+			await user.click(
+				screen.getByRole( 'button', { name: 'Unlink sides' } )
+			);
+
+			const slider = screen.getByRole( 'slider', {
+				name: 'Left and right sides',
+			} );
+
+			fireEvent.change( slider, { target: { value: 50 } } );
+
+			expect( slider ).toHaveValue( '50' );
+			expect(
+				screen.getByRole( 'textbox', { name: 'Top and bottom sides' } )
+			).not.toHaveValue();
+			expect(
+				screen.getByRole( 'textbox', { name: 'Left and right sides' } )
+			).toHaveValue( '50' );
+		} );
+
+		it( 'should show "Mixed sides" label when sides have different values but are linked', async () => {
+			const user = userEvent.setup();
+
+			render( <Example /> );
+
+			const unlinkButton = screen.getByRole( 'button', {
+				name: 'Unlink sides',
+			} );
+
+			await user.click( unlinkButton );
+
+			await user.type(
+				screen.getByRole( 'textbox', {
+					name: 'Right side',
+				} ),
+				'13'
+			);
+
+			expect(
+				screen.getByRole( 'textbox', { name: 'Right side' } )
+			).toHaveValue( '13' );
+
+			await user.click( unlinkButton );
+
+			expect(
+				screen.getByRole( 'textbox', { name: 'Mixed sides' } )
+			).toHaveValue( '' );
+		} );
```

Comment:
I added this test after updating the labels as I noticed there wasn't a test for 'Mixed sides'. 

I updated the label with a ternary so the label could be 'Mixed sides', but I'm curious to see what others think: 
https://github.com/WordPress/gutenberg/blob/3f9ec19bb0bc2e6d2c77f0305117215035c6d341/packages/components/src/box-control/all-input-control.tsx#L87

If we revert back to just `LABELS.all,` it would have the label 'All sides' with 'Mixed sides' as the placeholder.

Reply:
> If we revert back to just `LABELS.all,` it would have the label 'All sides' with 'Mixed sides' as the placeholder.

Interesting. I think it does makes more sense _without_ the ternary, because the `Mixed` placeholder signifies that we can't show a single value due to there being mixed values. Whereas, the `All sides` label signifies that if you input a value into the field, it will set that value to all sides.

Reply:
That makes sense! After seeing 'Mixed sides' as the placeholder, I think it might be confusing. So I'm thinking of going back to 'Mixed' or maybe 'Mixed values'?